### PR TITLE
Reorganizing project folder structure

### LINKED ISSUE #80: Culture is not supported
Hello i will run i take this error
image

### PR DESCRIPTION
**Pull request by Skrypt**
Reproduces the Orchard Core folder structure to be able to reuse the same Orchard Core source code files for building assets / docker images and else.

- [x] Fixing build with net5.0.
- [x] Adding vscode launch settings.
- [x] Adding Github Actions for PR's.
- [x] Make it multi-targetting netcoreapp3.1 and net5.0 to test with CI.
- [x] Fix Unit Tests and code that was failing with net5.0.
- [x] Adding functional tests for a ecommerce recipe.
- [x] Fix web project startup not launching browser from VS Code.

Fixes #80

### CODE REVIEW

**Comment by Skrypt** [comment]
Unit Tests failing under ubuntu latest. Seems like a Globalization culture issue.

File: test/OrchardCore.Commerce.Tests/CurrencyTests.cs [hunk 1]
```diff
@@ -25,8 +25,8 @@ public static CurrencyTheoryData TestData
         [MemberData(nameof(TestData))]
         public void CurrenciesProperlyFormatAmounts(ICurrency currency, decimal amount, string expectedFormat)
         {
-            var result = currency.ToString(amount);
-            Assert.Equal(expectedFormat, result);
+            var result = currency.ToString(amount).Replace(" ", "").Replace("￥", "¥");
+            Assert.Equal(expectedFormat.Replace(" ", "").Replace("￥", "¥"), result);
```
Comment (Skrypt) [inline test/OrchardCore.Commerce.Tests/CurrencyTests.cs:29]: This is kind of ugly but there are differences in currency formats between Windows and Ubuntu (Linux).

**Comment by Skrypt** [comment]
ECommerce theme needs some work. I copied the TheTheme from source code to start with.

**Comment by bleroy** [comment]
Thanks for the large contribution. Next time however, please try to break those down into smaller PRs instead of having one large PR with a checklist inside of it.

**Comment by Skrypt** [comment]
Thanks @bleroy I'm trying to get it working mostly. Fixing things broken as I find them. I removed the TODO list on this PR.
I could remove TheEcommerce theme for this PR but I think I want to keep it because I added functional tests on it. Of course, the theme is not ready but at least we have a starting point.

**Comment by bleroy** [comment]
I also wish we had discussed the change in structure before you put work into it. I haven't had time to look at the details yet, will provide feedback as soon as I can.

**Comment by Skrypt** [comment]
Yeah maybe we could make use of Nuget packages more here. I don't think we have any yet so yeah if you want this we need to setup a Nuget package repository first.

**Pull request closed without merging**